packages/firebase/index.d.ts

```diff
@@ -7758,10 +7758,18 @@ declare namespace firebase.firestore {
     /**
      * Whether to skip nested properties that are set to `undefined` during
      * object serialization. If set to `true`, these properties are skipped
-     * and not written to Firestore. If set `false` or omitted, the SDK throws
-     * an exception when it encounters properties of type `undefined`.
+     * and not written to Firestore. If set to `false` or omitted, the SDK
+     * throws an exception when it encounters properties of type `undefined`.
      */
     ignoreUndefinedProperties?: boolean;
+
+    /**
+     * Whether to merge the provided settings with the existing settings. If
+     * set to `true`, the settings will be merged with existing settings. If
```

Comment:
We avoid "will" when possible.  So this could be "are merged" and the line below just "settings replace..."